galaxyproject/galaxy
# Provide clues for advanced history search

**Why?**
To increase awareness and ease of access to Advanced Searching in Histories. 

**How?**
Added a button/link to Advanced Searching documentation to the input-query component, as an optional (default is null) value.

**How to Test:** 
1. Pull my branch
2. sh run.sh
3. Click on the ? icon in the Histories panel search box. Note how it takes you to the documentation.
4. Type input in and note how the text does not go behind the icons.
5. Notice how the search box in the tool panel does not have an icon, despite using a common component.  
6. Navigate to "view all histories". Note: All of the search input boxes here are are also equipped with Advanced Search button/link.  
7. Ask me if you have any questions. I appreciate feedback.
